_health-care/_js/_form.js

```diff
@@ -0,0 +1,348 @@
+var HealthApp = (function() {
+  "use strict";
+
+  var identityTransform = function(formData, arg) { return arg; }
```

Comment:
Might want to add a `;` at the end, though it probably doesn't matter because I think it gets automatically added anyway.